tests/test_ModelSerialization.py

```diff
@@ -187,3 +197,44 @@ def test_foreign_key_exposed(self, Classroom):
         )
         assert len(classroom) == 1
         assert classroom[0]['school_id'] == self.school.id
+
+    def test_serialize_order_by_single_column(self, Student):
+        students = Student.serialize(
+            to_return=['name'],
+            order_by=Student.name
+        )
+        assert len(students) == 2
+        assert students[0]['name'] <= students[1]['name']
+
+    def test_serialize_order_by_tuple(self, Student):
+        students = Student.serialize(
+            to_return=['name', 'email'],
+            order_by=(Student.name, Student.email)
+        )
+        assert students[0]['name'] <= students[1]['name']
+
+    def test_serialize_order_by_tuple_with_id(self, Student):
+        students = Student.serialize(
+            to_return=['name', 'email'],
+            order_by=(Student.name, Student.email, Student.id)
+        )
+        assert students[0]['name'] <= students[1]['name']
+
+    def test_serialize_with_limit_and_offset(self, Student):
```

Comment:
consider using teacher and return students.name - the interesting thing is that limit and offset work while fetching to many relationships (this is not trivial)